src/style.css

```diff
@@ -0,0 +1,128 @@
+.page-html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  color: #060b35;
+}
+
+.reset {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__img {
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__name {
+  padding-top: 40px;
+
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+}
+
+.product-card__code {
+  flex-basis: 100%;
+  padding-top: 4px;
+  width: 98px;
+
+  line-height: 14px;
+  font-size: 10px;
+
+  color: #616070;
+}
+
+.star-box {
+  display: flex;
+  flex-basis: 100%;
+  justify-content: space-between;
+  padding-top: 16px;
+}
+
+.star-box__stars {
+  display: flex;
+}
+
+.star-box__stars-star {
+  content: " ";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+
+  background-position: 50%;
+
+  height: 16px;
+  width: 16px;
+}
+
+.star-box__stars-star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star-box__reviews {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  line-height: 18px;
+  font-size: 10px;
+}
+
+.star-box__stars > .star-box__stars-star:last-child {
+  background-image: url(./images/star.svg);
+}
+
+.product-card__price {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding-top: 22px;
+
+  line-height: 18px;
+}
+
+.product-card__price-text {
+  color: #616070;
+}
+
+.product-card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.product-card__buy {
+  margin-top: 16px;
+  border-radius: 5px;
+  border: 1px solid transparent;
+  width: 164px;
+  height: 38px;
+  box-sizing: border-box;
+  
+  background: #00acdc;
+  color: #fff;
+
+  line-height: 38px;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
```

Comment:
Group your style properties - https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
> Group your style properties - https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

I can't understand what's wrong here

/* Display & Box Model */
  margin-top: 16px;
  border-radius: 5px;
  border: 1px solid transparent;
  width: 164px;
  height: 38px;
  box-sizing: border-box;
  
/* Color */
  background: #00acdc;
  color: #fff;

/* Text */
  line-height: 38px;
  font-weight: 700;
  font-size: 14px;
  text-align: center;
  text-decoration: none;
  text-transform: uppercase;